src/sentry/static/sentry/app/views/organizationEventsV2/modalLineGraph.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SentryTypes from 'app/sentryTypes';
+import {getInterval, useShortInterval} from 'app/components/charts/utils';
+import {getFormattedDate} from 'app/utils/dates';
+import EventsRequest from 'app/views/organizationEvents/utils/eventsRequest';
+import LineChart from 'app/components/charts/lineChart';
+import {Panel} from 'app/components/panels';
+import withApi from 'app/utils/withApi';
+import withGlobalSelection from 'app/utils/withGlobalSelection';
+
+const defaultGetCategory = () => t('Events');
+
+const ModalLineGraph = props => {
+  const {api, groupId, organization, location, selection} = props;
+
+  const isUtc = selection.datetime.utc;
```

Comment:
So if `selection.datetime.utc` is `null` it won't necessarily mean that the user doesn't want it to be formatted UTC, it just means that they haven't use the UTC override in the global selection header. I think we'd want to fallback to user's timezone in `ConfigStore`.